### ISSUE TITLE
AndHowUtil findConfiguration needs testing

### ISSUE THREAD

**Issue by eeverman** [eeverman/andhow#355]
Relatively simple to test, but requires some internal knowledge of how AndHow works to set it up.

**Comment by chace86** [comment]
@eeverman I would like to give this one a shot.

**Comment by chace86** [comment]
Never mind. This seems to be very difficult if not impossible to test in its current implementation.

**Comment by eeverman** [comment]
@chace86 Yes, this one is very tricky. Testing would involve simulating what a startup environment is like for AndHow. I will try to update my list of issues with some more bite sized tasks that don’t require as much background knowledge of the system.  
Thank you for you interest in AndHow.

**Comment by eeverman** [comment]
OBE - Lots of work was done in this area.
